app/src/main/java/io/appium/uiautomator2/model/internal/CustomUiDevice.java

```diff
@@ -119,7 +119,7 @@ public AccessibleUiObject findObject(Object selector) throws UiAutomator2Excepti
             node = (AccessibilityNodeInfo) selector;
             selector = toSelector(node);
         } else if (selector instanceof UiSelector) {
-            return toAccessibleUiObject(getUiDevice().findObject((UiSelector) selector), null);
```

Comment:
Can be removed...?

Reply:
yes. Basically providing null means we want to wait for the downstream method up to `Configurator.getInstance().getWaitForSelectorTimeout()` seconds if the given selector does not exist. This basically repeats the purpose of an implicit wait, which we have in the upstream driver. E.g. keeping null here would add unnecessary delays while looking for UiSelector's, which don't exist